### PR TITLE
PR SS-3 account creation page

### PR DESCRIPTION
**Pull request by qiuxiatang**
Set up the account creation page

tested no error

### CODE REVIEW

File: account_creation_page.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+with st.form("account_creation_form"):
+
+    st.header("Social Stock Analyzer")
+    st.subheader("Create your Account")
+
+    Username = st.text_input('',placeholder='Username')
+    Password = st.text_input('',placeholder='Password')
+    Confirm_password = st.text_input('',placeholder='Confirm password')
+    Email = st.text_input('',placeholder='E-mail address')
```
Comment (ojas1542) [inline account_creation_page.py]: We also need a field for first and last name.

Comment (qiuxiatang) [reply account_creation_page.py]: Ok, I will add them.

File: account_creation_page.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    st.subheader("Create your Account")
+
+    Username = st.text_input('',placeholder='Username')
+    Firstname = st.text_input('',placeholder='Firstname')
+    Lastname = st.text_input('',placeholder='Lastname')
+    Password = st.text_input('',placeholder='Password')
+    Confirm_password = st.text_input('',placeholder='Confirm password')
+    Email = st.text_input('',placeholder='E-mail address')
+
+    submitted = st.form_submit_button("Sign Up")
```
Comment (ojas1542) [inline account_creation_page.py:27]: Can you also add some logic to ensure the user doesn't leave any fields blank? Also if you can add enforcement of strong password guidelines(i.e 8 character minimum, uppercase, lowercase, numbers, and special chars) that would be great.

Comment (qiuxiatang) [reply account_creation_page.py:27]: Sounds good, I will deal with that.

File: account_creation_page.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import re
+
+
+with st.form("account_creation_form"):
+
+    st.header("Social Stock Analyzer")
+    st.subheader("Create your Account")
+
+    user_name = st.text_input('',placeholder='Username')
+    f_name = st.text_input('',placeholder='Firstname')
```
Comment (ojas1542) [inline account_creation_page.py]: Should be two words.

File: account_creation_page.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    l_name = st.text_input('',placeholder='Lastname')
```
Comment (ojas1542) [inline account_creation_page.py]: same here

**Review by ojas1542 — CHANGES_REQUESTED** [review]
Looking good. Just a few minor issues with naming. Also, if you could also have the error text be in red so it stands out more then we should be good to merge this.

File: account_creation_page.py (hunk 3 above)
Comment (qiuxiatang) [reply account_creation_page.py]: What do you mean, by two words? change f_name to first_name?

Comment (ojas1542) [reply account_creation_page.py]: it should prompt the user "First name" instead of "Firstname"

Comment (qiuxiatang) [reply account_creation_page.py]: I see. Thank you! I changed.

File: account_creation_page.py (hunk 4 above)
Comment (qiuxiatang) [reply account_creation_page.py]: Got it

**Review by ojas1542 — APPROVED** [review]
Looks great! Go ahead with the merge.

**Pull request merged**